src/context/types.ts

```
@@ -0,0 +1,30 @@
+    id: string;
+    type: 'group' | 'private';
+
+    send(content: string): Promise<void>;
+}
+
+export interface Content {
+    body: string;
+    media: Media | undefined;
+}
+
+export type Media = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)